src/Shared/E2ETesting/BrowserFixture.cs

```diff
@@ -52,9 +57,41 @@ public BrowserFixture()
             }
         }
 
+        public IWebDriver Browser { get; }
+
+        public ILogs Logs { get; }
+
+        public IMessageSink DiagnosticsMessageSink { get; }
+
+        public static bool HostSupportsBrowserAutomation => string.IsNullOrWhiteSpace(Environment.GetEnvironmentVariable("ASPNETCORE_BROWSER_AUTOMATION_DISABLED")) &&
+            (IsAppVeyor || (IsVSTS && RuntimeInformation.OSDescription.Contains("Microsoft Windows")) || OSSupportsEdge());
+
+        private static bool IsAppVeyor =>
+            Environment.GetEnvironmentVariables().Contains("APPVEYOR");
+
+        private static bool IsVSTS =>
+            Environment.GetEnvironmentVariables().Contains("TF_BUILD");
```

Comment:
Is this stuff still applicable? I didn't think we were using AppVeyor anywhere. And is VSTS another name of Azure Pipelines, or is thing another thing we could remove?

Also, the logic around "is Windows" and `OSSupportsEdge` seems redundant, since we're running these tests against Chrome, and they should be cross-platform now, right?

Reply:
No, there are a couple of small things I need to rationalize yet, but for the most part this brings everything to a single codebase

Reply:
I cleaned this up a bit.